src/lib/api-client.js

```diff
@@ -165,3 +165,10 @@ export function longTermCheckout(studentId, itemAddresses, adminCode, longtermDu
         longtermProfessor
     });
 }
+
+export function patchItemDueDate(studentId, address, date){
+    return patch('student/item', {studentId}, {
+        itemAddress: address,
```

Comment:
it's a query that will change the student not the address, but i need to know the itemAddress for the patch as well